### ISSUE TITLE
XMPP/Jabber community chat

### ISSUE THREAD

**Issue by mstenek** [bbatsov/projectile#1891]
Hello all:

The documentation lists a discord server, a google mailing list, and a freenode IRC channel as discussion venues. I offer another: [A Projectile xmpp/jabber public channel](xmpp:projectile@chat.disroot.org?join).

Why another one?

- Discord is privacy invasive: it requires a phone number to register (a big no no for me)
- The mailing list seems dead
- The freenode IRC channel doesn't exist

While registration to an xmpp/jabber service is required, some only ask for a username and password -- a far superior option than, say, discord. [Here's a list of public servers](https://list.jabber.at) available for account registration.

As for clients, [here's a good list from which to pick one](https://xmpp.org/software).

Hope to see you there!

**Comment by TABTHETEXTOUTFORWIMPS** [comment]
I completely agree with you that Discord is the absolute worst option for any FOSS project. 
 Doom Emacs also uses Discord and I refuse to join. Though I'd like to interact with the community threre, I'm just not willing to use Discord.  

XMPP, while private and distributed, can be quite difficult to use and can be unreliable when communicating cross-server.  Matrix may be a better, more modern option.  It supports E2E encryption in channels and while it does require an email to register, one can use mailinator or any other disposable email service to register.  Or, you can just set up your own Synapse sever and federate it with the network.  I think Matrix would probably be a better option, and there's a great Emacs community already there with many contributors who are very active.  My vote would be Matrix.

**Comment by bbatsov** [comment]
Discord was just inertia for me, as I already was using it for some other things. I don't really use chats much these days, and all the communication around the project tends to happen here. I'll probably just remove all other comm channels from the docs.
